ckanext/dcat/harvesters/rdf.py

```diff
@@ -401,70 +422,3 @@ def import_stage(self, harvest_object):
             model.Session.commit()
 
         return True
-        
-    def _parse_and_collect(
-        self,
-        items,
-        source_dataset,
-        harvest_job,
-        guids_in_source,
-        is_series=False,
-        collect_series_mapping=False
-    ):
-        object_ids = []
-        label = "dataset series" if is_series else "dataset"
-        series_mapping = {} if collect_series_mapping else None
-
-        for item in items:
-            original_title = item.get("title", label)
-            if not item.get("name"):
-                item["name"] = self._gen_new_name(original_title)
-
-            if item["name"] in self._names_taken:
-                suffix = len([i for i in self._names_taken if i.startswith(item["name"] + "-")]) + 1
-                item["name"] = f"{item['name']}-{suffix}"
-
-            self._names_taken.append(item["name"])
-
-            if not item.get("owner_org") and source_dataset.owner_org:
-                item["owner_org"] = source_dataset.owner_org
-
-            guid = self._get_guid(item, source_url=source_dataset.url)
-            if not guid:
-                self._save_gather_error(f"Could not get a unique identifier for {label}: {item}", harvest_job)
-                continue
-
-            item.setdefault("extras", []).append({"key": "guid", "value": guid})
-            guids_in_source.append(guid)
-
-            obj = HarvestObject(guid=guid, job=harvest_job, content=json.dumps(item))
-            obj.save()
-            object_ids.append(obj.id)
-
-            # Store mapping of RDF URI to dataset name if requested
-            if collect_series_mapping:
-                series_uri = item.get("uri") or item.get("identifier")
-                if series_uri:
-                    # Try to find an existing active dataset series by 'guid' match
-                    existing = model.Session.query(model.Package).\
-                        join(model.PackageExtra).\
-                        filter(model.PackageExtra.key == 'guid').\
-                        filter(model.PackageExtra.value == series_uri).\
-                        filter(model.Package.type == 'dataset_series').\
-                        filter(model.Package.state == 'active').\
-                        first()
-
-                    if existing:
-                        item["name"] = existing.name
-
-                    series_mapping[str(series_uri)] = {
-                        "id": existing.id if existing else item.get("id"),
-                        "name": item["name"]
-                    }
```

Comment:
Is this logic moved elsewhere? At first glance I don't see it in the scope of this PR